resources/benchmark/tpcds/all_queries/query_01.sql

```diff
@@ -0,0 +1,22 @@
+with customer_total_return as
+(select sr_customer_sk as ctr_customer_sk
```

Comment:
Could you run this through some SQL formatter? This is hard to read.

Reply:
Done.